src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -478,13 +480,13 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
 
     this.studentService.getStudentsFromCourse({ courseId: this.courseId }).subscribe(
     (resp: Students) => {
-      if (resp.students.length !== 0) {
-        this.loadExistingStudentsData(existingStudentsHOTInstance, resp.students);
-      } else {
+      if (resp.students.length === 0) {
         // Shows a message if there are no existing students. Panel would not be expanded.
         this.isExistingStudentsPresent = false;
         this.isExistingStudentsPanelCollapsed = !this.isExistingStudentsPanelCollapsed; // Collapse the panel again
+        return;
       }
```

Comment:
Not sure if this change is necessarily better than the if-else. What do you think?

Reply:
yes I agree